.upgrade-config.yml

```diff
@@ -1,2 +1,3 @@
 upstream-provider-name: terraform-provider-zitadel
-pr-assign: eliobischof
\ No newline at end of file
+target-version: v2.0.2
```

Comment:
@eliobischof you shouldn't set the target version in this file. After your permissions are fixed on this repo and you have configured the proper Github token, I'll give it a shot at updating the setup further. Don't just upgrade to v2 of your TF provider like all at once.